decode/render.ts

```diff
@@ -251,14 +254,14 @@ export function resize(data: ResizeData, iframe: HTMLIFrameElement): void {
 }
 
 export function change(data: InputChangeData, iframe: HTMLIFrameElement): void {
-    let el = element(data.target) as HTMLInputElement;
+    let el = element(data.target as number) as HTMLInputElement;
     if (el) { el.value = data.value; }
 }
 
 export function selection(data: SelectionData, iframe: HTMLIFrameElement): void {
```

Comment:
There's a design issue here. We really need not the start/end but the set of element in the selection. For a future PR.

Reply:
Yep, I see that. I'll fix it in next version where I am going to introduce Performance events and will have some schema changes.